inc/targetticket.class.php

```diff
@@ -35,6 +35,17 @@
 
 class PluginFormcreatorTargetTicket extends PluginFormcreatorTargetBase
 {
+
+   const TICKET_TYPE_INCIDENT = 1;
+   const TICKET_TYPE_REQUEST = 2;
+
+   public static function getEnumTicketType() {
+      return [
+         PluginFormcreatorTargetTicket::TICKET_TYPE_INCIDENT  => __('Incident', 'formcreator'),
```

Comment:
Why not reusing the constants of Ticket class ?

Reply:
changed